jis-api/src/main/java/pl/edu/uj/JImageStream/api/core/Filter.java

```diff
@@ -60,9 +60,11 @@ protected int getSourceWidth() {
     }
 
     public void setUp() {
+        logger.info("starting filter");
     }
 
     public void tearDown() {
+        logger.info("ending filter");
```

Comment:
same as above + we can add totaltime for filter here (I think there's a issue for that already)